partner_compassion/models/partner_compassion.py

```diff
@@ -113,6 +116,39 @@ def get_unreconciled_amount(self):
             res += move_line.credit
         return res
 
+    @api.multi
+    def _compute_children(self):
+        church = None
+        total = 0
+        for partner in self:
+            if partner.is_church:
+                church = partner
+            else:
+                partner.number_children = len(partner.sponsored_child_ids)
+                total += len(partner.sponsored_child_ids)
+        if church:
```

Comment:
Your way of doing it is wrong: If many churches are in `self`, you will only update the last one. Why not doing this directly in the for loop? Here you are adding all the the members of all churches and putting the result in the last church.